### PR TITLE
Update Rust version and dependencies.

### PR DESCRIPTION
**Pull request by afck**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1,17 +1,17 @@
 [[package]]
```
Comment (DrPeterVanNostrand) [inline Cargo.lock:1]: Do you want git to track this lock file?

File: Cargo.toml [hunk 2]
```diff
@@ -7,12 +7,12 @@ description = "Read POA voting records and rank voters by how many ballots they
...
+error-chain = { version = "0.12", default-features = false }
+ethabi = "6.0.1"
+ethabi-contract = "6.0.0"
+ethabi-derive = "6.0.2"
 parse_duration = "1.0.1"
 serde = "1.0.36"
 serde_derive = "1.0.36"
 serde_json = "1.0.13"
-web3 = { version = "0.3.0", default-features = false, features = ["http", "tls"] }
+web3 = { version = "0.4.0", default-features = false, features = ["http", "tls", "ws"] }
```
Comment (DrPeterVanNostrand) [inline Cargo.toml]: Did you add websockets to get around the http-only build error?

**Review by DrPeterVanNostrand — APPROVED** [review]

File: Cargo.lock (hunk 1 above)
Comment (afck) [reply Cargo.lock:1]: Yes, I think for binaries (as opposed to libraries) that's common: https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

File: Cargo.toml (hunk 2 above)
Comment (afck) [reply Cargo.toml]: Yes. I'll add a comment linking to https://github.com/tomusdrw/rust-web3/issues/157 before I merge. :+1:

**Pull request merged**